packages/test-ui/src/App.tsx

```diff
@@ -57,8 +57,8 @@ function App(): ReactElement | null {
 
   useEffect(() => {
     const init = async () => {
-      const json = await utils.fetchJson("https://raw.githubusercontent.com/connext/chaindata/main/crossChain.json");
-      setChainData(json);
+      const data = await getChainData();
+      setChainData(data);
```

Comment:
lol was this wrapped in an init just so we could use await? could just use `.then`